### ISSUE TITLE
cannot chdir to home directory of user nobody: No such file or directory

### ISSUE THREAD

**Issue by rhansen** [roehling/postsrsd#154]
When I start up PostSRSd I get the following in my logs:

```
postsrsd[8759]: cannot chdir to home directory of user nobody: No such file or directory
```

Wouldn't it be better to unconditionally `chdir` to `"/"`?  (That directory always exists, and it would prevent the process from interfering with `umount`.)

**Comment by roehling** [comment]
This warning pretty much only comes up with `nobody` as user, and I kind of like it, because it is much better to have a dedicated user anyway. I recently added support for [sysusers.d](https://github.com/roehling/postsrsd/commit/8f84b0da0291050a3ea7a1976be5f94301516238) which should make that pretty painless.

**Comment by rhansen** [comment]
The error message implies that the home directory's existence is important.  Is it?  Are relative paths in the config file relative to that home directory?

**Comment by roehling** [comment]
It is not an error, it is a warning. The home directory was initially supposed to be the anchor for relative paths, but I don't think it is really necessary nor desirable that way. I'll think about whether or not I want to drop that altogether.

**Comment by roehling** [comment]
Resolved in 2a003547ec1e45018cdcee29a8c78077c182604f
